.circleci/config.yml

```diff
@@ -8,7 +8,7 @@ jobs:
   toxify:
 
       machine:
```

Comment:
```suggestion
      docker:
```

Reply:
decided on updating the image instead of switching to docker, as later steps failed.

Reply:
decided on updating the image instead of switching to docker, as later steps failed.